tests/Unit/NumericalAlgorithms/LinearSolver/ResidualMonitorActionsTestHelpers.hpp

```diff
@@ -4,13 +4,16 @@
 #pragma once
 
 #include <cmath>
+#include <converse.h>
 #include <string>
 
 #include "DataStructures/DataBox/DataBox.hpp"
 #include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "IO/Observer/ReductionActions.hpp"
 #include "Utilities/Gsl.hpp"
 #include "Utilities/TMPL.hpp"
 #include "Utilities/TaggedTuple.hpp"
+#include "tests/Unit/IO/Observers/ObserverHelpers.hpp"
```

Comment:
I think this should include `ObservationId` instead

Reply:
I am including now "tests/Unit/ActionTesting.hpp" for the MockArrayChare and
I also include ObservationId.